### DISCUSSION TITLE
`World.GetFirstPlayerController()` returns `null` when `NetMode` set to client

### DISCUSSION THREAD

**Discussion by riddlemd** [nxrighthere/UnrealCLR#83] [Help] [answered]
![image](https://user-images.githubusercontent.com/272498/103143125-73856600-46d6-11eb-8486-e956aa1fce2b.png)

**Comment by nxrighthere** [comment, marked as answer]
This is the default behaviour of the engine. From my understanding as long as the client is not connected to the server, the controller doesn't persist in the world.
